java/lance-namespace-hms/src/main/java/com/lancedb/lance/namespace/util/PageUtils.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.lancedb.lance.namespace.util;
+
+import java.util.List;
+
+public class PageUtils {
```

Comment:
nit: `PageUtil`, just to match the pattern with `ValidationUtil`. Also this seems useful in general, should move to core.